### ISSUE TITLE
Signifier for Overlay Control

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-echo-show#106]
### Why/User Benefit/User Problem

Improve the user’s understanding of how the overlay works by adding a signifier to show it can be swiped up or down

### Acceptance Criteria (how do I know when I’m done?)

The homescreen card features a handle as a signifier and as an affordance that hints users that they can swipe up/down or drag up/down the homescreen card. This is possible also when the homescreen card features less than 4/8 tiles: [Mock](https://user-images.githubusercontent.com/1721875/46337872-ef46f400-c62e-11e8-8808-0c1c28587ca3.png)

**Comment by Baron-Severin** [comment]
@athomasmoz as currently implemented, down swipes on the handle or on the blank space above it will dismiss the overlay, but swipes starting on the overlay itself will not (see linked video).

https://photos.app.goo.gl/HY8u3fRudMgmzDvX9

Do you think this is a good experience, or would you like me to change it?

**Comment by athomasmoz** [comment]
@Baron-Severin Good catch, I agree it's not a great experience as is. How hard is it to change? I think it's worthwhile

**Comment by Baron-Severin** [comment]
@athomasmoz I don't think it should be too hard.  I'll look into it.

**Comment by Baron-Severin** [comment]
@athomasmoz I did some more work on this and ran into some problems.  The overlay becomes scrollable when the user has enough tiles, so if we allow down swipes to dismiss the overlay we will need some UX support to clarify how we distinguish between scrolls and swipes (swipe duration? velocity? initial touch location?).  There are also some technical challenges that might take another day or two to work out.

As a user, I was pretty happy with the behavior in the above video.  It seems pretty consistent and easy to use.  If it's alright with you, I would like to ship that and open a follow-up story for UX to define swipe vs scroll behavior.

**Comment by athomasmoz** [comment]
@Baron-Severin Thanks, I agree that sounds like a good approach

**Comment by mcomella** [comment]
> The overlay becomes scrollable when the user has enough tiles, so if we
allow down swipes to dismiss the overlay we will need some UX support to
clarify how we distinguish between scrolls and swipes (swipe duration?
velocity? initial touch location?).

I think the intention with the signifier is that you can only
swipe-to-dismiss the overlay by directly starting your swipe on the
signifier. However, depending on how it feels, we may want to increase the
size of the signifier so it's easier to trigger.
